src/board_controller/inc/brainflow_boards.h

```diff
@@ -222,13 +215,18 @@ json brainflow_boards_json = {
             {"num_rows", 10},
             {"eeg_channels", {1, 2, 3, 4, 5, 6, 7, 8}}
         }},
-        {"16",
```

Comment:
looks like you didnt resolve merge conflicts properly, in fact you removed one of supported boards... and there are more places like this

I think I will get commits from this PR, push them to another branch in brainflow-dev and fix such issues by myself when I get bitalino device

Reply:
when I add the Bitalino board there was not gforcepro in the repository, sorry for this conflict
we can fix it. Is 18 good for boardId?

Reply:
Yeah, 18 is fine. That would be great if you fix it. Create a PR to bitalino branch instead master

Reply:
Ok, I do it, now

Reply:
Thanks!